### PR TITLE
Add GH actions

### PR DESCRIPTION
**Pull request by cstamas**
To run also ITs

### CODE REVIEW

**Review by michael-o — APPROVED** [review]

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - name: Set up cache for ~/.m2/repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: maven-${{ matrix.os }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            maven-${{ matrix.os }}-
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
```
Comment (pzygielo) [inline .github/workflows/maven.yml]: why not current, maintained, v2?

Comment (cstamas) [reply .github/workflows/maven.yml]: yup, updating

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+      - name: Set up cache for ~/.m2/repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: maven-${{ matrix.os }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            maven-${{ matrix.os }}-
+
+      - name: Set up JDK 1.8
```
Comment (pzygielo) [inline .github/workflows/maven.yml:43]: https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md

needs 1.8 -> 8 changed,
and distribution selected

Comment (cstamas) [reply .github/workflows/maven.yml:43]: thanks @pzygielo for help! pls eyeball and approve if OK

**Review by pzygielo — APPROVED** [review]

**Pull request closed without merging**